vos/test/scripts/vospace-lock-atest.tcsh

```diff
@@ -121,12 +121,10 @@ foreach resource ($resources)
   if ( ${?TESTING_CAVERN} ) then
       echo " [SKIPPED, vos/issues/82]"
   else
-      echo "$LOCKCMD $CERT $CONTAINER $VUNLOCK_ARGS"
       $LOCKCMD $CERT $CONTAINER $VUNLOCK_ARGS > /dev/null || echo " [FAIL]" && exit -1
       echo " [OK]"
       echo -n "check unlocked container "
-      echo "$TAGCMD $CERT $CONTAINER $LIST_ARGS"
-      $TAGCMD $CERT $CONTAINER $LIST_ARGS | grep -q false && set SUCCESS = "true"
+      $TAGCMD $CERT $CONTAINER $LIST_ARGS | grep -q None && set SUCCESS = "true"
```

Comment:
The `grep -q false` changed to `grep -q None` in the `$TAGCMD` executions, but the `grep -q true` stayed the same in other execution checks. Is that consistent with the service behaviour?